testing/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(SOURCE_FILES 
+    testing.cpp
+    testing.h
+    test_float4.cpp)
+add_executable(testing ${SOURCE_FILES})
+target_link_libraries(testing tack)
```

Comment:
The "modern" syntax allows linked library propagation, its not required for this target as its an executable but it give you a bunch of other nice stuff like picking up the include directories from the `INTERFACE` header only library above.

```cmake
target_link_libraries(testing PRIVATE tack)
```

Here `PRIVATE` means only link `tack` into this library, not any other targets which link `testing`.